src/topology/subset_properties.lean

```diff
@@ -328,6 +328,11 @@ begin
   rw nhds_prod_eq, exact le_inf ha hb
 end
 
+/-- Finite topological spaces are compact. -/
+instance fintype.compact_space [fintype α] : compact_space α :=
```

Comment:
Aren't you supposed to use a lower priority, like 100, as this instance always applies? I guess the linter will complain.